test/e2e/e2e_suite_test.go

```diff
@@ -151,6 +152,7 @@ var _ = SynchronizedAfterSuite(func() {
 func initScheme() *runtime.Scheme {
 	sc := runtime.NewScheme()
 	framework.TryAddDefaultSchemes(sc)
+	Expect(certv1.AddToScheme(sc)).NotTo(HaveOccurred())
```

Comment:
Not sure, But I guess this might still be required.

Reply:
Shouldn't hurt to add it to the schemes.